packages/expo-device/src/ExpoDevice.web.ts

```diff
@@ -9,6 +9,10 @@ if (Platform.isDOMAvailable) {
   result = parser.getResult();
 }
 
+function convertGiBtoBytes(gib: number): number {
+  return gib * 1024 ** 3;
```

Comment:
According to the current spec, deviceMemory returns values that work well with IEEE 754 but if that were to change in the future, rounding away floating point inaccuracies is probably the right thing to do.
```suggestion
  return Math.round(gib * 1024 ** 3);
```